ci-operator/step-registry/vsphere/deprovision/customized-resourcepool/vsphere-deprovision-customized-resourcepool-commands.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+unset SSL_CERT_FILE
+unset GOVC_TLS_CA_CERTS
+INSTALL_CONFIG="${SHARED_DIR}/install-config.yaml"
+INFRA_ID=$(jq -r .infraID ${SHARED_DIR}/metadata.json)
+rsp_config=$(yq-go r ${INSTALL_CONFIG} 'platform.vsphere.failureDomains[*].topology.resourcePool')
+
+if [[ -z ${rsp_config} ]];then
+    echo "resource pool not defined in install-config. skip this step"
+    exit 0
+fi
+
+if govc pool.info -json ${rsp_config} | jq -r '.ResourcePools[].Vm[] | join(":")' | xargs govc ls -L | awk -F'/' '{print $NF}' | grep -q ${INFRA_ID};then
```

Comment:
I think here is two check points:
1. After clusters are destroyed, the resource pool still exists.
2. There is no vm in the resource pool.

It is better to run two separate if block to run the checks.

Reply:
ok